### DISCUSSION TITLE
Keep getting fetch is not a function

### DISCUSSION THREAD

**Discussion by hsb-tonmoy** [sveltejs/kit#6535] [Q&A]
```
// page.server.js

import { redirect } from '@sveltejs/kit';

export async function load({ params, locals, fetch }) {
	if (locals.user) {
		throw redirect(302, '/');
	}

	const key = params.key;

	const res = await fetch('http://127.0.0.1:8000/api/v1/auth/registration/verify-email/', {
		method: 'POST',
		headers: {
			'Content-Type': 'application/json'
		},
		body: JSON.stringify({ key })
	});

	if (res.ok) {
		return {
			message: 'success'
		};
	}

	return {
		message: 'invalid'
	};
}

```

![image](https://user-images.githubusercontent.com/35843150/188221599-5c782e9a-cceb-4ff7-9f2d-ec0ca98991d1.png)

This is a page.server.js file and the fetch keeps failing. I usually have an api.js that exports fetch but that wasn't working so I tried going this approach but it's not working either.

**Comment by infovore** [comment]
I believe that a `ServerLoadEvent` no longer has a `fetch` parameter; `fetch` is available on server-side routes without passing it in from the `ServerLoadEvent`. So try removing it from your `load` function signature and see what happens?
